scalafmt-tests/src/test/resources/scalajs/Class.stat

```diff
@@ -54,8 +54,7 @@ private class Encoder extends CharsetEncoder(
     }
 >>>
 object a {
-  private class Encoder
-      extends CharsetEncoder(UTF_16_Common.this, 2.0f, 2.0f,
+  private class Encoder extends CharsetEncoder(UTF_16_Common.this, 2.0f, 2.0f,
```

Comment:
Is this change intended?

Reply:
yes, same explanation is above. `always` attempts to binpack as much as possible unless there was an explicit break.